third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -331,6 +368,51 @@ function getStackFrames() {
   return callFrames;
 }
 
+
+window.DevOnly = {
+  // NOTE: commented out until we have a safe/reliably way of dealing with it
+  // tryEvalDev(expression, frameId = 0) {
+  //   // log(`[CHROMDEBUG] eval - expression: "${expression}"`);
+
+  //   // NOTE: expression sometimes gets wrapped in parentheses, and its value must be a string
+  //   const prefixes = ['("dev:', '"dev:'];
+  //   const prefix = prefixes.find(p => expression.startsWith(p));
+  //   if (prefix) {
+  //     // hackfix: evaluate straight-up in our dev context
+  //     // TODO: unsafe. Must be behind DEV-ONLY flag.
+
+  //     let cmd = expression;
+  //     if (cmd.startsWith('(')) {
+  //       // strip '()'
+  //       cmd = cmd.substring(1, expression.length - 1);
+  //     }
+  //     if (cmd.endsWith(';')) {
+  //       // strip trailing ';'
+  //       cmd = cmd.substring(0, expression.length - 1);
+  //     }
+
+  //     // parse JSON (used for serialization)
+  //     cmd = JSON.parse(cmd);
+
+  //     // strip "dev:" and wrap in ()
+  //     cmd = `(${cmd.substring(4)})`;
+
+  //     // run
+  //     const res = eval(cmd);
+  //     const resJson = JSON.stringify(res);
+
+  //     const t = res !== undefined ? ` (type: ${typeof res})` : '';
+  //     // log(`[CHROMDEBUG] eval (dev) - cmd: "${cmd}", res:${t} "${resJson}"`);
+
+  //     return { result: { data: {}, returned: { value: resJson } } };
+  //   }
+  // },
+
+  // popCommandError() {
+  //   return gLastCommandErrors.shift();
+  // }
+};
```

Comment:
If this is for debugging it should be removed unless we can get the functionality in permanently.

Reply:
(As I mentioned on Discord, I saw the comments too late, and kept on going. Providing the fixes on top of my next PR instead, which should go live today.)

Removed.

Link: [Here](https://github.com/replayio/chromium/commit/fbb71cbf6a9b9382c6fcf8b0f51d264b2e9eec5b).